Makefile

```diff
@@ -28,7 +34,9 @@ format-style:
 	autopep8 `git ls-files | grep "\.py$$"`
 
 test: clean check-syntax-errors check-style check-types
-	env PYTEST_ADDOPTS="$$PYTEST_ADDOPTS --cov=openfisca_core" pytest
```

Comment:
I don't know what it was used for locally, but are we sure we want to drop the `--cov=openfisca_core` option and now only run it in CI?

Reply:
Good catch! I think that change has little to do with this PR.